### PR TITLE
Update description of Cloud Native Glossary Repo

### PR DESCRIPTION
**Pull request by seokho-son**
Signed-off-by: Seokho Son <shsongist@gmail.com>

### Describe your changes
- Update description of Cloud Native Glossary Repository.
- Following Pinned repositories dashboard will be updated accordingly.
image

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

**Review by iamNoah1 — APPROVED** [review]
Awesome @seokho-son :)
lgtm

File: .github/settings.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ repository:
   name: glossary
 
   # A short description of the repository that will show up on GitHub
-  description: Cloud Native Glossary
+  description: The CNCF Cloud Native Glossary Project is to explain cloud native concepts in clear and simple language without requiring any previous technical knowledge. (https://glossary.cncf.io/)
```
Comment (CathPag) [inline .github/settings.yml:9]:
```suggestion
  description: The CNCF Cloud Native Glossary Project aims to define cloud native concepts in clear and simple language, making them accessible to anyone —  whether they have a technical background or not (https://glossary.cncf.io).
```

**Review by CathPag — COMMENTED** [review]
Great catch! A few edits. Let me know what you think.

File: .github/settings.yml (hunk 1 above)
Comment (seokho-son) [reply .github/settings.yml]: Thanks @CathPag 
Your suggestion lgtm :)
I directly applied your modification.

**Review by CathPag — APPROVED** [review]
LGTM :)

**Pull request merged**